<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateVoiceCommandWidget</class>
 <widget class="QWidget" name="CreateVoiceCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Action:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbAction"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Visible Trigger:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KPushButton" name="pbFromTemplate">
     <property name="text">
      <string>Create from Action template</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leVisibleTrigger"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbShowIcon">
     <property name="text">
      <string>Show icon</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
